Assets/GothicVR/Scripts/Creator/Meshes/AbstractMeshCreator.cs

```diff
@@ -492,14 +499,18 @@ protected virtual Texture2D GetTexture(string name)
             return AssetCache.I.TryGetTexture(name);
         }
 
-        protected Material GetDefaultMaterial()
+        protected Material GetDefaultMaterial(bool isAlphaTest)
         {
-            var standardShader = Shader.Find(defaultShader);
-            var material = new Material(standardShader);
-
-            // Enable clipping of alpha values.
-            material.EnableKeyword("_ALPHATEST_ON");
-
+            var shader = Shader.Find(defaultShader);
+            if (isAlphaTest)
+            {
+                shader = Shader.Find("Unlit/Unlit-AlphaToCoverage");
```

Comment:
As we don't reference shaders inside scene, we also need this shader to be added to Project Settings --> Graphics --> Always Includes Shaders